README.md

```diff
@@ -1 +1,2 @@
-# vba-migrator-addin
+## Upload file to drive
+Code itself is self-explanatory
```

Comment:
This is not encouraged. Please explain what this code does in brief and you can point to the link from where you have got the code pointers.

Reply:
Done